### PR TITLE
event history tab in activities page

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-08 at 12 20 14 am

**Comment by liviavinci** [comment]
screen shot 2018-03-08 at 12 59 10 am

File: src/scenes/Activities/scenes/EventHistory/index.js [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    return (
+      <Grid container spacing={0}>
+        {
+          transactions.length ?
+            (<Table>
+              {this.getTableHeader()}
+              {this.getTableRows(this.state.transactions)}
+            </Table>) :
+            (<Typography variant="body1">
+              You do not have any transactions right now.
```
Comment (dwalintukan) [inline src/scenes/Activities/scenes/EventHistory/index.js]: localization

File: src/scenes/Activities/scenes/EventHistory/index.js [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+            </Typography>)
+        }
+      </Grid>
+    );
+  }
+
+  executeTxsRequest() {
+    this.props.getTransactions([], undefined);
```
Comment (dwalintukan) [inline src/scenes/Activities/scenes/EventHistory/index.js]: you need to add filters for this one. need to add all of the types except for `Transfer` type

File: src/scenes/Activities/scenes/EventHistory/index.js [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    const transactions = order === 'desc'
+      ? this.state.transactions.sort((a, b) => (b[orderBy] < a[orderBy] ? -1 : 1))
+      : this.state.transactions.sort((a, b) => (a[orderBy] < b[orderBy] ? -1 : 1));
+
+    this.setState({ transactions, order, orderBy });
+  }
+
+  getEventURL(topicAddress, oracleAddress) {
+    // TODO: NEED TO REROUTE TO TOPIC PAGE IS ITS IN WITHDRAW STATUS
+    return `/oracle/${topicAddress}/${oracleAddress}`;
```
Comment (dwalintukan) [inline src/scenes/Activities/scenes/EventHistory/index.js]: also need to handle `Withdraw` txs too. Withdraw txs are actually done on the TopicEvent itself (not an Oracle).

File: src/scenes/Activities/scenes/EventHistory/index.js [hunk 4]
```diff
@@ -218,6 +232,10 @@ class EventHistory extends React.Component {
   }
 
   getEventURL(topicAddress, oracleAddress) {
+    if (!oracleAddress) {
```
Comment (dwalintukan) [inline src/scenes/Activities/scenes/EventHistory/index.js]: prefer to check if `!_.isEmpty(topicAddress)` instead of just assuming topicAddress is there.

**Comment by dwalintukan** [comment]
🚢

**Pull request merged**